# Circular arc section.
Point2D center
float32 radius
float32 start_angle
float32 end_angle
uint32 color